Code/src/main/resources/ui/style.css

```diff
@@ -129,3 +208,9 @@ LineChart {
     -fx-font-weight: bold;
     -fx-background-color: rgba(0, 0, 0, 0.3);
 }
+
+.invisTouchZone {
+}
+
+.leftArgument {
+}
```

Comment:
These last two don't seem to add much :)

By the way, it seems the whole file has a kind of mixture of tabs and spaces going on - could you fix that?